tests/cephfs/nfs/subvolume_export.py

```diff
@@ -0,0 +1,155 @@
+        client1 = clients[0]
+        fs_util.prepare_clients(clients, build)
+        fs_util.auth_list(clients)
+        mon_node_ip = fs_util.get_mon_node_ips()
+        mon_node_ip = ",".join(mon_node_ip)
+        rhbuild = config.get("rhbuild")
+        nfs_servers = ceph_cluster.get_ceph_objects("nfs")
+        nfs_server = nfs_servers[0].node.hostname
+        nfs_name = "cephfs-nfs"
+        nfs_export_1 = "/export_" + "".join(
+            secrets.choice(string.digits) for i in range(3)
+        )
+        nfs_export_2 = "/export_" + "".join(
+            secrets.choice(string.digits) for i in range(3)
+        )
+        fs_name = "cephfs"
+        nfs_mount_1 = "/mnt/nfs_" + "".join(
+            secrets.choice(string.ascii_uppercase + string.digits) for i in range(5)
+        )
+        nfs_mount_2 = "/mnt/nfs_" + "".join(
+            secrets.choice(string.ascii_uppercase + string.digits) for i in range(5)
+        )
+        subvol_grp = "subvol_group_nfs_1"
+        subvol_1 = "subvol_nfs_1"
+        subvol_2 = "subvol_nfs_2"
+        commands = [
+            f"ceph fs subvolume create {fs_name} {subvol_1}",
+            f"ceph fs subvolumegroup create {fs_name} {subvol_grp}",
+            f"ceph fs subvolume create {fs_name} {subvol_2} {subvol_grp}",
+        ]
+        for command in commands:
+            client1.exec_command(sudo=True, cmd=command, long_running=True)
+        subvol_1_path, rc = client1.exec_command(
+            sudo=True, cmd=f"ceph fs subvolume getpath {fs_name} {subvol_1}"
+        )
+        subvol_2_path, rc = client1.exec_command(
+            sudo=True,
+            cmd=f"ceph fs subvolume getpath {fs_name} {subvol_2} {subvol_grp}",
+        )
+        if "5.0" in rhbuild:
+            client1.exec_command(
+                sudo=True,
+                cmd=f"ceph nfs export create cephfs {fs_name} {nfs_name} "
+                f"{nfs_export_1} path={subvol_1_path}",
+            )
+            client1.exec_command(
+                sudo=True,
+                cmd=f"ceph nfs export create cephfs {fs_name} {nfs_name} "
+                f"{nfs_export_2} path={subvol_2_path}",
+            )
+        else:
+            client1.exec_command(
+                sudo=True,
+                cmd=f"ceph nfs export create cephfs {nfs_name} "
+                f"{nfs_export_1} {fs_name} path={subvol_1_path}",
+            )
+            client1.exec_command(
+                sudo=True,
+                cmd=f"ceph nfs export create cephfs {nfs_name} "
+                f"{nfs_export_2} {fs_name} path={subvol_2_path}",
+            )
+        rc = fs_util.cephfs_nfs_mount(client1, nfs_server, nfs_export_1, nfs_mount_1)
+        if not rc:
+            log.error("cephfs nfs export mount failed")
+            return 1
+        rc = fs_util.cephfs_nfs_mount(client1, nfs_server, nfs_export_2, nfs_mount_2)
+        if not rc:
+            log.error("cephfs nfs export mount failed")
+            return 1
+        commands = [
```

Comment:
Hi @yogesh-mane - We have a function for running IOs, can you replace this set of commands with the run_io function ??

Reply:
Hi @yogesh-mane,
can mark this as resolve Conversation.
if the comments are addressed

Reply:
Sure @AmarnatReddy